packages/loot-design/src/components/modals/CreateLocalAccount.js

```diff
@@ -117,15 +117,31 @@ function CreateLocalAccount({ modalProps, actions, history }) {
                     justifyContent: 'flex-end',
                   }}
                 >
-                  <label style={{ userSelect: 'none' }}>
+                  <label style={{
+                    userSelect: 'none',
+                    textAlign: 'right'
+                  }}>
                     <input
                       name="offbudget"
                       type="checkbox"
                       checked={!!values.offbudget}
                       onChange={handleChange}
                       onBlur={handleBlur}
                     />{' '}
-                    Off-budget
+                    <span> Off-budget </span>
+                    <div style={{ fontSize: '0.7em', color: 'grey' }}>
```

Comment:
I _think_ I have addressed all of these. I've moved it out of the `label` (TIL!) into a `Text` and replaced the color to use the same grey used in the main window behind this panel. 

I am not entirely sure about the 
> generally avoid raw `<div>s`

As the only way to get this to align was to put the `Text` block inside a `<div>`, but this may be me not knowing front end at all 🙈, so happy to be educated!